### ISSUE TITLE
What is the correct way to redraw a control?

### ISSUE THREAD

**Issue by jlopezr** [VladimirMarkelov/clui#144]
I am using a label for showing information that I receive from an asynchronous goroutine through a channel. When I receive new data, I modify the title of the label and then use the ui.RefreshScreen() function. 

This is working for me but I am not sure if this redraws all the screen or only the modified control.

What is the expected way to ask the library to redraw a specific control?

**Comment by VladimirMarkelov** [comment]
Yes, it is the correct way. There are two ways: do not call anything and it redraws after the next key or mouse event or, if you need to do it immediately, you have to call `ui.RefreshScreen` somehow. On the one hand, it seems overkill to refresh the screen if you need to redraw a single element; on the other hand, internally it should optimize out redundant screen output and result in repainting only changed area in the screen
